change/@fluentui-react-calendar-compat-acdf3646-9c24-46d9-85eb-7f68e84c551d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31509&commitId=a839c3c03a1d1dc9f087a1e0cb58fde0c97c2eac&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Calendar Compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar Compat.Default.default.chromium.png|1612| Changed|
|Calendar Compat.Custom icons.default.chromium.png|1612| Changed|
|Calendar Compat.Default - RTL.default.chromium.png|1612| Changed|

</details>


  <details><summary><b>DatePicker Compat</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- work week.default.chromium.png|1680| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|2510| Changed|
|DatePicker Compat.DateRange- day.default.chromium.png|1612| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|1593| Changed|
|DatePicker Compat.marked dates.default.chromium.png|1626| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|1665| Changed|
|DatePicker Compat.default.opened.chromium.png|1612| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|1743| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|1722| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|2494| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|2074| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|1767| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|2053| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|2290| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|1687| Changed|
|DatePicker Compat.when rendering inline, it should not render behind relatively positioned elements.default.chromium.png|1612| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Removed|
|SwatchPicker Converged.default.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Calendar changes look good to me, the others I think are from previous PRs(?)

Reply:
I think, yes. didn't change anything else. DatePicker has expected changes as it uses CalendarPicker